packages/commands/tests/src/index.spec.ts

```diff
@@ -1153,7 +1153,7 @@ describe('@lumino/commands', () => {
         document.body.removeEventListener('keydown', keydown);
       });
 
-      it('should ignore modifier keys pressed in the middle of key sequence', () => {
+      it('should register a key sequence', () => {
```

Comment:
This test and the following should probably not been changed.
IIUC, these modifications will break the key sequence containing modifiers.

Reply:
As mod keys now have additional functionality, pressing a mod key will break the sequence when entering a key sequence i.e. [Shift K, Shift L]. 

The test 'should ignore modifier keys pressed in the middle of key sequence' has been changed to reflect this. A similar change has been made to 'should process key sequences that use different modifier keys' to remove unnecessary keystrokes.

'should return nothing for keys that are marked as modifier in keyboard layout' has been modified as mod keys now return keyboard events.